### PR TITLE
Modify update description to better describe patchable entities

### PR DESCRIPTION
**Pull request by mooreds**
Patchable entities should have a better update section blurb. This PR extracts out a good explanation and applies to all entities that are patchable.

Partially addresses: https://github.com/FusionAuth/fusionauth-issues/issues/1283

* This extraction was already done for entities, so that was rolled in.
* Not sure about the double variable in _oauth-idp-operations.adoc, so need to test that.

### CODE REVIEW

File: site/docs/v1/tech/apis/integrations.adoc [hunk 1]
```diff
@@ -47,7 +47,9 @@ include::docs/v1/tech/apis/_integration-response-body.adoc[]
 
 == Update Integrations
 
-This API is used to update the global System Configuration object.
+:capitalized_object_name: Integration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```
Comment (robotdan) [inline site/docs/v1/tech/apis/integrations.adoc:52]: This API won't require an Id on the URL.

File: site/docs/v1/tech/apis/registrations.adoc [hunk 2]
```diff
@@ -127,7 +127,9 @@ include::docs/v1/tech/apis/_user-registration-response-body.adoc[]
 
 == Update a User Registration
 
-This API is used to update a User Registration for a User. This API is used to update the User's registration for a specific Application. You must specify the User Id for the User that is updating their information on the URI. The Id of the Application the User is updating their information for is sent in the request body. This API doesn't merge the existing User Registration and your new data. It replaces the existing User Registration with your new data.
+:capitalized_object_name: User Registration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```
Comment (robotdan) [inline site/docs/v1/tech/apis/registrations.adoc:132]: Will this generic explanation cover this API? There isn't a registration Id, so the unique Id is the `userId` + `applicationId`.

File: site/docs/v1/tech/apis/system.adoc [hunk 3]
```diff
@@ -63,7 +63,9 @@ include::docs/v1/tech/apis/_system-configuration-response-body.adoc[]
 
 == Update the System Configuration
 
-This API is used to update System Configuration.
+:capitalized_object_name: System Configuration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```
Comment (robotdan) [inline site/docs/v1/tech/apis/system.adoc:68]: This API will not have a unique Id on the URL.

**Review by robotdan — COMMENTED** [review]
Made a few small comments, otherwise LGTM.

File: site/docs/v1/tech/apis/integrations.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/integrations.adoc:53]: Thanks, handled.

File: site/docs/v1/tech/apis/registrations.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/registrations.adoc:133]: Changed to handle this composite key.

File: site/docs/v1/tech/apis/system.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/system.adoc:69]: Thanks, handled.

**Pull request merged**